sdk/eventhub/Azure.Messaging.EventHubs/src/Producer/EventDataBatch.cs

```diff
@@ -49,11 +49,28 @@ public sealed class EventDataBatch : IDisposable
         ///
         private TransportEventBatch InnerBatch { get; }
 
+        /// <summary>
+        ///   The fully qualified Event Hubs namespace that the batch is associated with.  To be used
+        ///   during instrumentation.  This is likely to be similar to
```

Comment:
(extreme) nit:  You can probably remove the friendly "This is likely similar...." since we're not exposing this.